libraries/AP_Camera/AP_Camera_Backend.h

```diff
@@ -138,6 +138,7 @@ class AP_Camera_Backend
     uint32_t last_photo_time_ms;    // system time that photo was last taken
     Location last_location;         // Location that last picture was taken at (used for trigg_dist calculation)
     uint16_t image_index;           // number of pictures taken since boot
+    bool is_recording = false;      // true if recording is on
```

Comment:
```suggestion
    bool is_recording;              // true if recording is on
```
Don't need to init false